CSharp/DevVmPowershell/Helpers/Constants.cs

```diff
@@ -20,5 +20,28 @@ public class Agents
 			public const int AGENT_INTERVAL = 20;
 			public const Agent.LoggingLevelEnum AGENT_LOGGING_LEVEL = Agent.LoggingLevelEnum.All;
 		}
+
+		public class CommonFields
+		{
+			public const string ControlNumber = "Control Number";

```

Comment:
Can we create a subclass Documents and move it there? or Rename it to DocumentCommandFields just to be specific.